src/rule/icmp.rs

```diff
@@ -64,6 +77,7 @@ impl IcmpType {
             EchoRep => 0,
             Unreach(_) => 3,
             EchoReq => 8,
+            Timex(_) => 11,
```

Comment:
When I wrote testing I mostly meant to actually run the library and create rules on your real system. And verify that the rules turned out as expected via `pfctl -sr`. Did you do this? For such a system level library I find it easier to verify it works by writing an `example/` binary instead of a unit test. Sometimes both makes sense.

I guess we can merge these unit tests. But I think they are more maintenance burden than what they provide benefits. It's just a duplication of all codes in a different format.

Reply:
> When I wrote testing I mostly meant to actually run the library and create rules on your real system.

That makes a lot more sense :) I agree that these unit tests are a maintenance burden. When I was writing them I was thinking "OK, I guess it could help catch a typo".

> And verify that the rules turned out as expected via `pfctl -sr`. Did you do this?

I did do this but I have not but it under the `example/` binary. I can do that as well as remove this redundant unit tests.

Reply:
Here is the output after my change

```
sudo pfctl -a test.anchor -sr
No ALTQ support in kernel
ALTQ related functions disabled
pass all no state
pass quick inet all no state
pass on utun0 inet6 all no state
block drop inet from 10.0.0.0/8 to any
pass inet proto icmp all icmp-type echoreq no state
pass inet proto icmp all icmp-type unreach code port-unr no state
pass inet proto icmp all icmp-type timex code transit no state
pass inet proto icmp all icmp-type timex code reassemb no state
```